src/Tests/Microsoft.Diagnostics.Monitoring.ExecuteActionApp/Microsoft.Diagnostics.Monitoring.ExecuteActionApp.csproj

```diff
@@ -1,10 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net6.0</TargetFrameworks>
```

Comment:
You need to update https://github.com/dotnet/dotnet-monitor/blob/6b1993792fdb0a00e9bfe06790b60a8cf20a2c3d/src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/ExecuteActionTestHelper.cs#L17 to use `Net60` instead of `NetCoreApp31`.